tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/create/CreateApplicationOnBuilderImageTest.java

```diff
@@ -127,7 +128,11 @@ private void validateJBIDE22704() {
 		String gitUrl = new LabeledText("Git Repository URL:").getText();
 
 		assertNotNull(gitUrl);
-		assertTrue("Git URL is empty (JBIDE-23704)", gitUrl.length() > 0);
+		if (gitUrl.length() < 1){
+			new CancelButton().click();
```

Comment:
This is already done here: https://github.com/jbosstools/jbosstools-openshift/blob/master/test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/wizard/AbstractOpenShiftApplicationWizard.java
On line 71 (case of validateJBIDE22704FromShellMenu) or on line 115 (case of validateJBIDE22704FromCentral)